uint8 freeze_toggle=1
---
string status
